restcomm/restcomm.http/src/main/java/org/mobicents/servlet/restcomm/http/AccountsEndpoint.java

```diff
@@ -235,6 +249,23 @@ protected Response putAccount(final MultivaluedMap<String, String> data, final M
                     account = account.setRole(parent.getRole());
                 }
                 dao.addAccount(account);
+
+                // Create default SIP client data
+                MultivaluedMap<String, String> clientData = new MultivaluedMapImpl();
+                clientData.add("Login", data.getFirst("EmailAddress"));
+                clientData.add("Password", data.getFirst("Password"));
+                clientData.add("FriendlyName", account.getFriendlyName());
+                clientData.add("AccountSid", account.getSid().toString());
+
+                Client client = clientDao.getClient(clientData.getFirst("Login"));
+                if (client == null) {
+                    client = createClientFrom(account.getSid(), clientData);
+                    clientDao.addClient(client);
+                } else if (!client.getAccountSid().toString().equals(account.getSid().toString())) {
+                    return status(CONFLICT)
```

Comment:
In the case where the client is already in use, Restcomm shouldn't respond with error message for the account creation. You should simply ignore it and move, or log something